components/textfield/metadata/textarea.yml

```diff
@@ -174,8 +174,18 @@ examples:
         <textarea id="textarea-keyboard-focused-invalid" name="field"  class="spectrum-Textfield-input" pattern="[\w\s]+" required>Invalid input</textarea>
       </div>
 
+  - id: textfield-grow
+    name: TextArea with Grow Behavior
+    description: Use `spectrum-Textfield--grows` to disable the drag behavior along with appropriate JavaScript to enable a text area that grows with input.
+    markup: |
+      <div class="spectrum-Textfield spectrum-Textfield--multiline spectrum-Textfield--grows">
+        <label for="textfield-helptext" class="spectrum-FieldLabel spectrum-FieldLabel--sizeM">Password</label>
+        <textarea id="textfield-helptext" name="field" class="spectrum-Textfield-input"></textarea>
+      </div>
+
   - id: textfield-quiet
     name: Quiet
+    description: Text areas should rarely be presented in the quiet style. The height of the field should use grow logic so the height will adjust with the height of the entered value.
```

Comment:
> The height of the field should use grow logic so the height will adjust with the height of the entered value.

I'm not clear on what this means. Do we expect this to require JS implementation?